app/views/shared/_sidenav.html.erb

```diff
@@ -43,7 +43,7 @@
         </ul>
       <%- end -%>
     </li>
-    <li><%= link_to 'Entitlements', page_path('entitlements'), class: current_page?(id: 'entitlements') ? 'usa-current' : nil %></li>
+    <li><%= link_to 'Entitlements', page_path('entitlements'), class: controller_name == 'entitlements' ? 'usa-current' : nil %></li>
```

Comment:
Oh wait, there is a `current_page?` defined already. Can that be updated or reused?